components/button/index.js

```diff
@@ -26,11 +26,24 @@ class Button extends Component {
 	}
 
 	render() {
-		const { href, target, isPrimary, isLarge, isToggled, className, disabled, ...additionalProps } = this.props;
+		const {
+			href,
+			target,
+			isPrimary,
+			isSecondary,
+			isLarge,
+			isSmall,
+			isToggled,
+			className,
+			disabled,
+			...additionalProps, // eslint-disable-line comma-dangle
```

Comment:
Without the `eslint-disable-line` here, `eslint` reports this line as an error.  This seems like a bug.